components/AppBar/AppBar.test.js

```diff
@@ -0,0 +1,5 @@
+import AppBar from './AppBar'
+
+it('renders without crashing', () => {
+  shallow(<AppBar />)
+})
```

Comment:
There should be an expect here for this test to be valid.
```suggestion
  const wrapper = shallow(<AppBar />);
  expect(wrapper.find(AppBar).length).toEqual(1);
})
```